lib/util.js

```diff
@@ -91,7 +91,7 @@ const showNextSteps = config => {
     );
   } else {
     log(
-      chalk`{green.bold SSH key and config generated successfully 🎉  and public key has been printed to the console. Please copy it to your Clipboard. \n\n}`
+      chalk`{green.bold SSH key and config generated successfully 🎉  and public key has been printed to the console(check output above). Please copy it to your Clipboard. \n\n}`
```

Comment:
This workaround implemented to tell user to see for public key output in console because of ordering issues. I'll fix it sooner in next version.